pytest-embedded/pytest_embedded/plugin.py

```diff
@@ -682,6 +684,17 @@ def _fixture_classes_and_options(
                             'part_tool': part_tool,
                         }
                     )
+            elif 'arduino' in _services:
+                from pytest_embedded_arduino.app import ArduinoApp
+
+                classes[fixture] = ArduinoApp
+                kwargs[fixture].update(
+                    {
+                        'pexpect_proc': pexpect_proc,
```

Comment:
After moving `build_dir` to the base `App` class, we could move these `kwargs` all to the default one

```python
    if fixture == 'app':
        kwargs['app'] = {'app_path': app_path, 'build_dir': build_dir}
```